xgboost_ray/main.py

```diff
@@ -382,6 +384,8 @@ class RayParams:
     # Distributed callbacks
     distributed_callbacks: Optional[List[DistributedCallback]] = None
 
+    verbose: bool = False
```

Comment:
```suggestion
    verbose: bool = True
```

can we default this to True for non-tune runs and to False for tune runs (which includes Ray AIR)?

Reply:
Hmm, how do you do that?